src/components/editor/property/InputListLOC.jsx

```diff
@@ -25,6 +25,17 @@ class InputListLOC extends Component {
     }
   }
 
+  // From https://github.com/ericgio/react-bootstrap-typeahead/issues/389
+  onKeyDown = (e) => {
```

Comment:
Can this code be extracted and shared among all 3 components?

Reply:
It could, but (1) these components need to be refactored in general to remove redundancy and (2) we don't share component action functions anywhere else, so this would be a one-off.

If you feel strongly, I will extract but otherwise my gut is to leave as is.

Reply:
I'm fine with it as is;  Sandi Metz told us:

- repeat code once -- hold your nose and do it
- repeat code twice - still hold your nose and do it, esp. if code has low churn